arangod/RocksDBEngine/RocksDBOptionFeature.h

```diff
@@ -172,6 +172,7 @@ class RocksDBOptionFeature final : public ArangodFeature,
   bool _partitionFilesForPrimaryIndexCf;
   bool _partitionFilesForEdgeIndexCf;
   bool _partitionFilesForVPackIndexCf;
+  bool _partitionFilesForZkdIndexCf;
```

Comment:
It looks like this variable is not initialized anywhere, so it has an undefined startup value.